pkg/apis/kops/cluster.go

```diff
@@ -349,6 +352,18 @@ type ClusterSubnetSpec struct {
 	Type SubnetType `json:"type,omitempty"`
 }
 
+type EgressProxySpec struct {
+	HTTPProxy     HTTPProxySpec `json:"httpProxy,omitempty"`
+	ProxyExcludes string        `json:"excludes,omitempty"`
+}
+
+type HTTPProxySpec struct {
```

Comment:
Can we abstract this to a Proxy Object, and then reuse for seperate HTTP and HTTPS?